js/main.js

```diff
@@ -16,6 +29,92 @@ $newEntryForm.addEventListener('submit', event => {
   newEntry.entryID = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(newEntry);
-  $entryImg.src = 'images/placeholder-image-square.jpg';
+  $newEntryImg.src = 'images/placeholder-image-square.jpg';
   $newEntryForm.reset();
+
+  // Create dom tree and add new entry to the page
+  $entryList.prepend(renderEntry(newEntry));
+  viewSwap('entries');
+
+  // Hide no entries text
+  if ($entryList.firstChild) {
+    toggleNoEntries($formView);
+  }
+});
+
+// Render a dom tree for entries
+function renderEntry(entry) {
+  const $entryItem = document.createElement('li');
+  $entryItem.setAttribute('class', 'row');
+
+  const $entryCol1 = document.createElement('div');
+  $entryCol1.setAttribute('class', 'entry-col');
+
+  const $entryImg = document.createElement('img');
+  $entryImg.setAttribute('src', entry.photoURL);
+  $entryImg.setAttribute('class', 'entry-img');
+  $entryImg.setAttribute('alt', entry.title);
+
+  const $entryCol2 = document.createElement('div');
+  $entryCol2.setAttribute('class', 'entry-col');
+
+  const $entryH2 = document.createElement('h2');
+  $entryH2.textContent = entry.title;
+
+  const $entryNote = document.createElement('p');
+  $entryNote.textContent = entry.notes;
+
+  $entryItem.appendChild($entryCol1);
+  $entryItem.appendChild($entryCol2);
+  $entryCol1.appendChild($entryImg);
+  $entryCol2.appendChild($entryH2);
+  $entryCol2.appendChild($entryNote);
+
+  return $entryItem;
+}
+
+// Determine what content to show on the page at load
+document.addEventListener('DOMContentLoaded', event => {
+  for (const entry in data.entries) {
+    $entryList.appendChild(renderEntry(data.entries[entry]));
+  }
+
+  if (data.view === 'entries') {
+    viewSwap('entries');
+  } else if (data.view === 'entry-form') {
+    viewSwap('entry-form');
+  }
```

Comment:
Instead of doing a conditional here, you should just call `viewSwap` utilizing your `data.view`.